cpu/sam0_common/periph/rtt.c

```diff
@@ -51,6 +51,12 @@ static void _wait_syncbusy(void)
 #endif
 }
 
+static void _rtt_read_req(void)
+{
+    RTC->MODE0.READREQ.reg = RTC_READREQ_RREQ;
+    _wait_syncbusy();
```

Comment:
```suggestion
#ifdef RTC_READREQ_RREQ
    RTC->MODE0.READREQ.reg = RTC_READREQ_RREQ;
    _wait_syncbusy();
#endif
```

Reply:
Oh its only a samd21 thing. Thanks for the catch!